stored_requests/backends/file_fetcher/fetcher.go

```diff
@@ -5,126 +5,133 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"path"
+	"path/filepath"
 	"strings"
 
 	"github.com/prebid/prebid-server/stored_requests"
 )
 
-// NewFileFetcher _immediately_ loads stored request data from local files.
-// These are stored in memory for low-latency reads.
-//
-// This expects each file in the directory to be named "{config_id}.json".
-// For example, when asked to fetch the request with ID == "23", it will return the data from "directory/23.json".
+// NewFileFetcher lazy-loads various kinds of objects from the given directory
+// Expected directory structure:
+// - stored_requests/{id}.json for stored requests
+// - stored_imps/{id}.json for stored imps
+// - {adserver}.json non-publisher specific primary adserver categories
+// - {adserver}/{adserver}_{account_id}.json publisher specific categories for primary adserver
 func NewFileFetcher(directory string) (stored_requests.AllFetcher, error) {
-	storedData, err := collectStoredData(directory, FileSystem{make(map[string]FileSystem), make(map[string]json.RawMessage)}, nil)
-	return &eagerFetcher{storedData, nil}, err
+	_, err := ioutil.ReadDir(directory)
```

Comment:
Nitpick: Since `err` is short lived, consider writing as:
```
if _, err := ioutil.ReadDir(directory); err != nil {
  return &fileFetcher{}, err
}
```